week-5/C-promises/exercise.js

```diff
@@ -44,7 +60,15 @@ function exercise3() {
   of your promise and insert it into the #exercise4 element
 */
 function exercise4() {
-  var promise4 = concatPromise()
+  var promise4 = concatPromise();
+    promise4.then(result => {
+      let newResult = result + 'Than';
+      document.querySelector('#exercise4').textContent = newResult;
```

Comment:
This exercise is correct! Maybe you can look at how this one works to solve exercises 1, 2 and 3

Reply:
Thank you very much for your time. I will need help. However, I am on holiday for almost 3 weeks from Tuesday. I will try my best to improve it!

Reply:
No problem :) I'm also around on slack if you need some more help